src/scrapers/hapoalim.ts

```diff
@@ -112,6 +113,16 @@ function convertTransactions(txns: ScrapedTransaction[]): Transaction[] {
   });
 }
 
+function getStartDate(options: ScraperOptions) {
+  const maximumYearsBack = 2;
+  const defaultStartMoment = moment().subtract(maximumYearsBack, 'years').add(1, 'day');
```

Comment:
all the scrapers default time is 1 years, changing it here will cause a breaking change. please remove any changes not related to the original issue from this PR. Thanks :)

btw in #728 we discussed about introducing a new option to allow customization of that default start time. but that PR is also not for that.

If you want please create an **issue** about ability to customize that value and we can discuss there the API adjustments

Reply:
This was part of the issue since 1 year was hardcoded in the startDate logic in the hapoalim scraper that's why I don't understand how that affects other scrapers and how it's a breaking change?

Reply:
This will enable me to extend scraping to 2 years for Hapoalim but being it a global config, it will be confusing as one may think this option affect all scrapers in addition to the fact that it won't allow setting different values for each scraper.

I think that a better and a more straight forward approach will be to implement a new per scraper config level with the current startDate and defaultStartMoment values for backward compatibly.
This will enable passing different values, flags, option which are scraper specific and also refactor away the code duplicity we have now where each scraper defines it's own defaultStartMoment.
@esakal wdyt?